feature/src/main/resources/feature.xml

```diff
@@ -52,38 +52,56 @@ limitations under the License.
         <bundle>mvn:org.apache.jclouds.api/atmos/${jclouds.version}</bundle>
```

Comment:
Latest version of namespace is `http://karaf.apache.org/xmlns/features/v1.4.0`. `resolver` attribute is no longer used so can drop it.